packages/teams-js/test/private/files.spec.ts

```diff
@@ -36,7 +37,7 @@ describe('files', () => {
   describe('getCloudStorageFolders', () => {
     it('should not allow calls before initialization', async () => {
       await expect(() => files.getCloudStorageFolders('channelId', emptyCallback)).toThrowError(
-        'The library has not yet been initialized',
+        new Error(errorLibraryNotInitialized),
```

Comment:
Just out of curiosity - it's interesting to me that these tests were previously calling `toThrowError` and passing a string vs an `Error`. Does Jest internally do the same sort of comparison? Or did we used to throw a `string` and now we are throwing an `Error`?

Reply:
toThrowError does some magic under the covers looking for Errors that match the supplied Error OR errors with the same message as the provided string